arrows/colmap/tests/test_colmap.cxx

```diff
@@ -0,0 +1,73 @@
+#include <arrows/colmap/colmap.h>
+#include <colmap/mvs/image.h>
+#include <fstream>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <test_gtest.h>
+#include <vital/algo/image_io.h>
+#include <vital/plugin_loader/plugin_manager.h>
+#include <vital/types/image.h>
+#include <vital/types/image_container.h>
+
+char* logo_path;
```

Comment:
We should put in the file path to the test image relative to the test directory. Take a look at how it is done in the ocv arrow - [here](https://github.com/Kitware/kwiver/blob/11bce47e962ed6577b6425887522bbcd54e98132/arrows/ocv/tests/test_inpaint.cxx#L20-L22),  [here](https://github.com/Kitware/kwiver/blob/11bce47e962ed6577b6425887522bbcd54e98132/arrows/ocv/tests/test_inpaint.cxx#L36),  [here](https://github.com/Kitware/kwiver/blob/11bce47e962ed6577b6425887522bbcd54e98132/arrows/ocv/tests/test_inpaint.cxx#L46), and [here](https://github.com/Kitware/kwiver/blob/11bce47e962ed6577b6425887522bbcd54e98132/arrows/ocv/tests/test_inpaint.cxx#L83). You will need to copy the test image to `data` in the test directory.